tasks/benchmark.rake

```diff
@@ -9,7 +9,11 @@ def bm_files
 end
 
 def build_config_name
-  MRUBY_TARGET
+  if !ENV['MRUBY_CONFIG'].to_s.empty?
+    File.basename(ENV['MRUBY_CONFIG'], '.rb').gsub('build_config_', '')
+  else
+    "build"
```

Comment:
I am not sure the rationale behind this change. Why "build"?
Could you explain?

Reply:
I just put it back (I'm not sure intention of original author).

https://github.com/mruby/mruby/commit/fdbfeaf533fd75143f5d3af3ec6c585b9da746c4#diff-3c833bf555cc95c834d1765cffde8d995db0413412d33785fc0ea872cc74946a

However, in `Rakefile`, `ENV['MRUBY_CONFIG']` equated unspecified with empty, but that wasn't the case here, so I've unified it.